docs/Documentation/Drivers/Classic/FluxOutputFiles.md

```diff
@@ -17,28 +17,26 @@ Each output file contains model output from one grid cell. The files use the nam
 | Nlayer+14                                                                                                         	| OUT_EVAP_CANOP  	| mm                       	| Evaporation from canopy storage                                     	|
 | Nlayer+15                                                                                                         	| OUT_TRANSP_VEG  	| mm                       	| Transpiration from the vegetation                                   	|
 | Nlayer+16                                                                                                         	| OUT_EVAP_BARE   	| mm                       	| Evaporation from bare soil                                          	|
 | Nlayer+17                                                                                                         	| OUT_SUB_CANOP   	| mm                       	| Sublimation from canopy interception                                	|
 | Nlayer+18                                                                                                         	| OUT_SUB_SNOW    	| mm                       	| Sublimation from ground snow pack                                   	|
-| The following four variables are only output when FULL_ENERGY or FROZEN_SOIL is TRUE in the global parameter file 	|                 	|                          	|                                                                     	|
+| The following four variables are only output when `FULL_ENERGY` or `FROZEN_SOIL` is TRUE in the global parameter file 	|                 	|                          	|                                                                     	|
 | Nlayer+19                                                                                                         	| OUT_SENSIBLE    	| W/m2                     	| Sensible heat flux from the surface                                 	|
 | Nlayer+20                                                                                                         	| OUT_GRND_FLUX   	| W/m2                     	| Ground heat flux plus heat storage in the top soil layer            	|
-| Nlayer+21                                                                                                         	| OUT_DELTAH      	| W/m2 (ALMA_OUTPUT: J/m2) 	| Rate of change in heat storage                                      	|
+| Nlayer+21                                                                                                         	| OUT_DELTAH      	| W/m2  	| Rate of change in heat storage                                      	|
 | Nlayer+22                                                                                                         	| OUT_FUSION      	| W/m2                     	| Net energy used to melt/freeze soil moisture                        	|
```

Comment:
make sure this lines up with the other output variables (it looks like it doesn't)